docs/_includes/attrs-env.adoc

```diff
@@ -67,6 +67,11 @@ For example, if the backend is `docbook5`, the basebackend is `docbook`.
 |Set if content is being converted to an embeddable document (body only).
 |
 
+|figure-number
+|The number of the most recent formal (ie titled) figure.
+You can set it to any integer value to change the subsequent figure numbers.
```

Comment:
Let's not state that this attribute can be changed. Instead, let's emphasize that it's auto-generated and should thus be treated as a reserved name.